### PR TITLE
WC-930 Use native <select> for single item selects

### PR DESCRIPTION
**Pull request by Klaster1**
https://ggsystems.atlassian.net/browse/WC-930

### CODE REVIEW

File: modules/web-console/frontend/app/primitives/form-field/index.scss [hunk 1]
```diff
@@ -276,6 +267,10 @@
                 right: 36px;
             }
         }
+        & > select.ng-invalid.ng-touched:not([multiple]) {
+            background-position: calc(100% - 36px) 50%;
+            padding-right: calc(28px + 26px);
```
Comment (AlexDel) [inline modules/web-console/frontend/app/primitives/form-field/index.scss:271]: @Klaster1 Why `calc` in `padding-right` if we know the exact pixels number? Won't just `padding-right: 54px` do the thing?

File: modules/web-console/frontend/app/primitives/form-field/index.scss [hunk 2]
```diff
@@ -377,6 +372,10 @@
                 right: 10px;
             }
         }
+        & > select.ng-invalid.ng-touched:not([multiple]) {
+            background-position: calc(100% - 10px) 50%;
+            padding-right: calc(28px + 26px);
```
Comment (AlexDel) [inline modules/web-console/frontend/app/primitives/form-field/index.scss:376]: @Klaster1 Please, look at the comment above.

**Review by AlexDel — CHANGES_REQUESTED** [review]
@Klaster1 Everything looks good for me. Just two css related issue. Please relate.

File: modules/web-console/frontend/app/primitives/form-field/index.scss (hunk 1 above)
Comment (Klaster1) [reply modules/web-console/frontend/app/primitives/form-field/index.scss:271]: 28px and 26px are used elsewhere too, I wanted to convey that, but was too lazy to extract the all the values into variables.

Comment (AlexDel) [reply modules/web-console/frontend/app/primitives/form-field/index.scss:271]: Ok. I see.

**Review by AlexDel — APPROVED** [review]

**Pull request closed without merging**